app/controllers/accounts_controller.rb

```diff
@@ -524,6 +524,7 @@ def update
         end
 
         params[:account][:turnitin_host] = validated_turnitin_host(params[:account][:turnitin_host])
+        params[:account][:ally_base_url] = validated_ally_base_url(params[:account][:ally_base_url])
```

Comment:
Can we make this the host instead of the full url? I don't want administrators controlling the plugin to be in charge if it will be HTTP or HTTPS.

Presumably we can have a deployment environment variable of some sort that could say if we want it to be secure or insecure for dev environments. turnitin's terminology indicates that this is probably what they do as well.